site/content/docs/monitoring/whitelisting.md

```diff
@@ -82,3 +82,12 @@ const page = await browser.newPage({userAgent})
 ```
 {{< /tab >}}
 {{< /tabs >}}
+
+## User Agent used by Checkly
```

Comment:
@josep-fl this is just a suggestion: I would call this "Default Checkly user agent"